minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMain.java

```diff
@@ -85,8 +85,6 @@ public static void main(String[] args) {
     public static void printValidateUsage() {
         System.out.println("Validate Usage:");
         System.out.println();
-        System.out.print("java ");
```

Comment:
Hey @apiri, I chose to remove them because they can lead to very confusing errors messages for user who ran it using the scripts. If they happen to mis-type a file location not only would they see the error saying the file doesn't exist but also this "java <class name> ..." line that was quite confusing.

Also, I see this being run using the script in >99% of the cases.